data/json/npcs/factions.json

```diff
@@ -649,6 +649,15 @@
     "food_supply": 87500,
     "wealth": 82500,
     "relations": {
+      "gods_community": {
+        "kill on sight": false,
+        "watch your back": true,
+        "share my stuff": true,
+        "guard your stuff": true,
+        "lets you in": true,
+        "defends your space": true,
+        "knows your voice": true
+      },
       "free_merchants": {
         "kill on sight": false,
         "watch your back": true,
```

Comment:
watch your back": true,
        "share my stuff": true,
        "guard your stuff": true,
        "defends your space": true,

These ones specifically should all be false.

Reply:
If they are true the they are effectively the same faction.  These should be changed for tacoma commune as well.